molecule/default/tests/test_default.py

```diff
@@ -42,24 +40,27 @@ def test_files2(host, f):
 def test_version_and_license(host):
     """Verify that Cobalt Strike is licensed and is an expected version."""
     cmd = host.run("cd /opt/cobaltstrike && ./teamserver")
-    regex = r"^\[\*\] Team Server Version: (?P<version>\d+\.\d+\.\d+) \((?P<year>\d{4})(?P<month>\d{2})(?P<day>\d{2})\) (?P<licensed>.*)$"
+    regex = r"^\[\*\] Team Server Version: (?P<version>(\d+\.)?\d+) (?P<licensed>.*)$"
```

Comment:
Thoughts on something like this? I fear that they will release a `4.8.1` or similar release which breaks the regex here. As an example please see [pip 23.0](https://pypi.org/project/pip/23.0/) and [pip 23.0.1](https://pypi.org/project/pip/23.0.1/).

```suggestion
    regex = r"^\[\*\] Team Server Version: (?P<version>(\d+)(\.\d+){1,2}) (?P<licensed>.*)$"
```

Reply:
That makes sense to me.